service/src/test/java/uk/gov/hmcts/reform/fpl/service/job/UpcomingHearingsContentProviderTest.java

```diff
@@ -25,6 +25,7 @@
 import static java.util.Optional.ofNullable;
 import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
 import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.reform.fpl.enums.TabUrlAnchor.ORDERS;
 
 @ExtendWith(SpringExtension.class)
```

Comment:
vv minor but seeing as we are editing this file this can be replaced with MockitoExtension instead of SpringExtension

Reply:
caused some problems so just leaving it for now, can cleanup later